daemon/mgr/container.go

```diff
@@ -1020,6 +1006,12 @@ func (mgr *ContainerManager) Update(ctx context.Context, name string, config *ty
 	}
 	c.Config.Env = newEnvSlice
 
+	if mgr.containerPlugin != nil {
+		if err = mgr.containerPlugin.PostUpdate(c.BaseFS, c.Config.Env); err != nil {
+			return err
```

Comment:
Do we have a rule to record the error log in the caller's code or the implementation part? @HusterWan

Reply:
Yes, we will record the error log in the end of api.